tests/Aspire.Dashboard.Tests/Integration/Playwright/PlaywrightFixture.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Workload.Tests;
+using Microsoft.Playwright;
+using Polly;
+using Polly.Retry;
+using Xunit;
+
+namespace Aspire.Dashboard.Tests.Integration.Playwright;
+
+public class PlaywrightFixture : IAsyncLifetime
+{
+    public IBrowser Browser { get; set; } = null!;
+    private ResiliencePipeline _resiliencePipeline = null!;
+
+    public async Task InitializeAsync()
+    {
+
+        var retryOptions = new RetryStrategyOptions
+        {
+            ShouldHandle = new PredicateBuilder().Handle<ArgumentException>(),
+            BackoffType = DelayBackoffType.Exponential,
+            UseJitter = true,
+            MaxRetryAttempts = 20,
+            Delay = TimeSpan.FromMilliseconds(100),
+        };
+
+        var resiliencePipelineBuilder = new ResiliencePipelineBuilder();
+        resiliencePipelineBuilder
+            .AddRetry(retryOptions)
+            .AddTimeout(TimeSpan.FromSeconds(10));
+        _resiliencePipeline = resiliencePipelineBuilder.Build();
+
+        Browser = await PlaywrightProvider.CreateBrowserAsync();
+    }
+
+    public async Task DisposeAsync()
+    {
+        await Browser.CloseAsync();
+    }
+
+    public async Task GoToHomeAndWaitForDataGridLoad(IPage page)
+    {
+        await page.GotoAsync("/");
```

Comment:
I recommend page -> Page like we have in our base classes.

```suggestion
        await Page.GotoAsync("/");
```

Reply:
As a method parameter, that does not follow naming conventions. If `page` were a property, I would agree

Reply:
I see, thought its a property on the class where page gets created in TestInitialize etc. Feel free to ignore then.